src/application/tools/prepare-cluster/__tests__/prepare-cluster.test.ts

```
@@ -168,9 +169,10 @@
 
     it('should handle session service updateAtomic errors', async () => {
       const session = createMockSession();
-      const mockGet = mockContext.sessionService!.get;
-      mockGet.mockResolvedValue(session);
-      const mockUpdateAtomic = mockContext.sessionService!.updateAtomic;
+      const sessionService = mockContext.sessionService as any;
+      sessionService.get.mockResolvedValue(session);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)